### DISCUSSION TITLE
What is the future roadmap for discordpy?

### DISCUSSION THREAD

**Discussion by nabtron** [Rapptz/discord.py#8440] [General]
Hi. I want to confirm the future roadmap for the library? Should we continue coding in it or start moving the bots to discordjs?

Thanks.

**Comment by lol219** [comment]
@nabtron what do you mean, discord.py is already working and revived

**Comment by lol219** [comment]
And it will be maintained in the future.

**Comment by nabtron** [comment]
@lol219 i mean the plan for support? roadmap, timeline, or will it get abandoned one fine day out of the blues like it was done in past?

**Reply by lol219** [reply]
> @lol219 i mean the plan for support? roadmap, timeline, or will it get abandoned one fine day out of the blues like it was done in past?

@nabtron i mean, the owner said that  he will keep maintaining discord.py in the future, which means that  the library is still alive and still getting developed, and he didn't said anything about the discontinuation when he resumed the development of dpy.

**Comment by nabtron** [comment]
@lol219 can you please give link to where he said that he will keep maintaining it?

**Reply by lol219** [reply]
> @lol219 can you please give link to where he said that he will keep maintaining it?

https://discord.com/channels/336642139381301249/336642776609456130/1005890760626933810
If you are not on dpy, then discord.gg/dpy.
And he already said on the gist of resuming that he will continue devving.
@nabtron

**Comment by nabtron** [comment]
i joined that now, but can't see any "roadmap" ?
